### DISCUSSION TITLE
Profile 7 to 8 muxing

### DISCUSSION THREAD

**Discussion by bw171291** [quietvoid/dovi_tool#322] [Q&A]
Just a quick question. 

If I do dovi_tool deluxe profile7.hevc

I get both a bl.hevc and el.hevc.

Then if I do dovi_tool -m 2 mux --bl.hevc --el.hevc, will that resulting hevc file have the extended layer baked into it? Or will profile 8.1 still ignore the extended layer?

**Comment by quietvoid** [comment]
> Then if I do dovi_tool -m 2 mux --bl.hevc --el.hevc, will that resulting hevc file have the extended layer baked into it? Or will profile 8.1 still ignore the extended layer?

The resulting profile 8.1 will ignore the EL regardless of what you do.
